docs/source/reference/straxen.analyses.rst

```diff
@@ -4,6 +4,30 @@ straxen.analyses package
 Submodules
```

Comment:
I always thought this got auto-updated but that appears not. Thanks for adding it!

Reply:
Yes, I was also a bit confused about it.